docs/developers/reference/api/linea-getproof.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: linea_getProof
+description: Reference content for the linea_getProof method.
+image: /img/socialCards/lineagetproof.jpg
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# `linea_getProof`
+
+Returns the account and storage values, including the Merkle proof, of the specified account.
+The supplied block parameter must be an L2 block that has been finalized on L1.
+
+:::info important
+
+Linea currently does not support the block parameter `finalized`, meaning that to get the current
+finalised block on the L1, users must retrieve the value from the `currentL2BlockNumber` variable in
+the [LineaRollup L1 contract](https://etherscan.io/address/0xd19d4b5d358258f05d7b411e21a1460d11b0876f#readProxyContract).
+:::
+
+## Parameters
+
+- `address`: A string representing the address (20 bytes) to check for balance.
```

Comment:
`to check for balance` seems too restricted for the use of account balance check (a proof can verify other account values like nonce, codehash, etc..), maybe we could say in a more general way like `for which the proof is requested`?